### PR TITLE
Update bootswatch auto-update config

### PR DESCRIPTION
**Pull request by m5o**
fix bootswatch npm auto-update
* https://github.com/thomaspark/bootswatch
* https://www.npmjs.com/package/bootswatch

🔗 https://github.com/thomaspark/bootswatch/issues/766

# Profile of the lib
 * Git repository (required): https://github.com/thomaspark/bootswatch
 * NPM package url (optional): https://www.npmjs.com/package/bootswatch

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/bootswatch/package.json [hunk 1]
```diff
@@ -16,10 +16,9 @@
   "npmName": "bootswatch",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "dist",
       "files": [
-        "fonts/*",
-        "*/*.css"
+        "*.css"
```
Comment (sufuf3) [inline ajax/libs/bootswatch/package.json]: Let's use `**/!(*.scss)`
Because the pattern `*.css` can't get the files which under subfolder.
Thanks

**Comment by m5o** [comment]
@sufuf3 Done

**Comment by sufuf3** [comment]
@m5o May I ask you how do you send this pull request? via GitHub website or use git command?
I think the commit should squash to one commit, if you use git command, we can help you how to do this.
But if you use GitHub website, it can't squash the commit. I can help you squash to one commit.

BTW, the commit message need to be modified to 
```
Update bootswatch auto-update config
```

Many thank.

**Comment by m5o** [comment]
@sufuf3 changes were made via the github website. Because this repository is just to damn uge to check it out. It take ages and it’s almost imposible to just change directories. 😡 

Please read about this blog post: https://github.com/blog/2141-squash-your-commits
It’s not an april fools day joke, it’s just a nice feature you should enable and use. 
Then, instead of request more ping pong tasks just merge and change the commit msg to whatever you like.

**Comment by sufuf3** [comment]
@m5o I'm really sorry for bother you.
I know that, but I just follow the reviewer's rule. 
If the contributor could help squash the commits, it is a great work.
But if no, than I can help do this.

This is my part of intern training that I need to ask the contributor if he/she can do this, else I can help squash the commit.

Really sorry for bother you.
I'll help you squash the commits to one commit.

Sorry and thank you for your great work.

**Comment by PeterDaveHello** [comment]
Thanks @m5o

**Comment by thezoggy** [comment]
just fyi, bootswatch 3.4.0 came out two days ago but cdnjs still has not picked it up.

**Comment by PeterDaveHello** [comment]
as you can see the update config was changed in this PR, it doesn't match the old directory structure

**Comment by thezoggy** [comment]
ty!

**Pull request merged**